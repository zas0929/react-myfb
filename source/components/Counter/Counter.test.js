import React from "react";
import Counter from "./";
import renderer from "react-test-renderer";

const renderTree = renderer.create(<Counter count={3} />).toJSON();

test("Counter should match snapshot", () => {
  expect(renderTree).toMatchSnapshot();
});
