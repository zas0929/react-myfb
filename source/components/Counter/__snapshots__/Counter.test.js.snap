// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter should match snapshot 1`] = `
<section>
  Posts count: 
  3
</section>
`;
